### PR TITLE
style,refactor:(Swiched all dropdown menus using datalist to html select with apropreate css. and made API key SensitiveInput)

### PR DESCRIPTION
**Pull request by snehinsen**
# Pull Request Checklist

### Note to first-time contributors: Please open a discussion post in [Discussions](https://github.com/open-webui/open-webui/discussions) and describe your changes before submitting a pull request.

**Before submitting, make sure you've checked the following:**

- [x ] **Target branch:** Please verify that the pull request targets the `dev` branch.
- [ ] **Description:** Provide a concise description of the changes made in this pull request.
- [x ] **Changelog:** Ensure a changelog entry following the format of [Keep a Changelog](https://keepachangelog.com/) is added at the bottom of the PR description.
- [ ] **Documentation:** Have you updated relevant documentation [Open WebUI Docs](https://github.com/open-webui/docs), or other documentation sources?
- [ ] **Dependencies:** Are there any new dependencies? Have you updated the dependency versions in the documentation?
- [x ] **Testing:** Have you written and run sufficient tests for validating the changes?
- [ x] **Code review:** Have you performed a self-review of your code, addressing any coding standard issues and ensuring adherence to the project's coding standards?
- [ ] **Prefix:** To cleary categorize this pull request, prefix the pull request title, using one of the following:
  - **BREAKING CHANGE**: Significant changes that may affect compatibility
  - **build**: Changes that affect the build system or external dependencies
  - **ci**: Changes to our continuous integration processes or workflows
  - **chore**: Refactor, cleanup, or other non-functional code changes
  - **docs**: Documentation update or addition
  - **feat**: Introduces a new feature or enhancement to the codebase
  - **fix**: Bug fix or error correction
  - **i18n**: Internationalization or localization changes
  - **perf**: Performance improvement
  - **refactor**: Code restructuring for better maintainability, readability, or scalability
  - **style**: Changes that do not affect the meaning of the code (white-space, formatting, missing semi-colons, etc.)
  - **test**: Adding missing tests or correcting existing tests
  - **WIP**: Work in progress, a temporary label for incomplete or ongoing work

# Changelog Entry

### Description

Replaced all instances of 
``` 
<input
	list="..."
	class="..."
	bind:value={...}
	placeholder="PLACEHOLDER"
/>

<datalist id="...">
	...
</datalist>
```
with

```
<select
	class="..."
	bind:value={...}
>
	<option value="" disabled selected>.PLACEHOLDER</option>
	// OPTIONS ITERTED HERE
</select>
```

### Changed

A simple UI edit for the way dropdowns for STT and TTS work, and made API key for Elevenlabs a SensitiveInput for security

### CODE REVIEW

File: src/lib/components/admin/Settings/Audio.svelte [hunk 1]
```diff
@@ -402,7 +401,7 @@
 						<div class=" mb-1.5 text-sm font-medium">{$i18n.t('TTS Model')}</div>
 						<div class="flex w-full">
 							<div class="flex-1">
-								<input
+								<!-- <input
```
Comment (tjbck) [inline src/lib/components/admin/Settings/Audio.svelte:404]: Same here.

File: src/lib/components/admin/Settings/Audio.svelte [hunk 2]
```diff
@@ -444,7 +452,7 @@
 							<div class=" mb-1.5 text-sm font-medium">{$i18n.t('TTS Voice')}</div>
 							<div class="flex w-full">
 								<div class="flex-1">
-									<input
+									<!-- <input
```
Comment (tjbck) [inline src/lib/components/admin/Settings/Audio.svelte:455]: Should be reverted.

File: src/lib/components/admin/Settings/Audio.svelte [hunk 3]
```diff
@@ -455,15 +463,24 @@
...
+									</select>
 								</div>
 							</div>
 						</div>
 						<div class="w-full">
 							<div class=" mb-1.5 text-sm font-medium">{$i18n.t('TTS Model')}</div>
 							<div class="flex w-full">
 								<div class="flex-1">
-									<input
+									<!-- <input
```
Comment (tjbck) [inline src/lib/components/admin/Settings/Audio.svelte:483]: Same here.

**Comment by tjbck** [comment]
Only the elevenlabs option should be updated to use select tag. Please squash the commits and feel free to reopen!

**Comment by snehinsen** [comment]
I did that with every single TTS and STT dropdown so...

**Pull request closed without merging**